src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/AddCacheWarmerPassTest.php

```diff
@@ -21,7 +21,7 @@ public function testThatCacheWarmersAreProcessedInPriorityOrder()
         $services = array(
             'my_cache_warmer_service1' => array(0 => array('priority' => 100)),
             'my_cache_warmer_service2' => array(0 => array('priority' => 200)),
-            'my_cache_warmer_service3' => array(),
+            'my_cache_warmer_service3' => array(0 => array()),
```

Comment:
had to fix this in the tests because the container method wouldn't return like this, same goes for the other test.